File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.py
def solution(id_pw, db):
    if id_pw in db:
        return 'login'
    id_list = []
    for i in range(len(db)):
        id_list.append(db[i][0])
    if id_pw[0] not in id_list:
        return 'fail'
    else:
        return 'wrong pw'
    return answer

File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.java
import java.util.Arrays;

class Solution {
    public String solution(String[] id_pw, String[][] db) {
        int n = db.length;
        for (String[] row : db) {
            if (Arrays.equals(row, id_pw)) {
                return "login";
            }
        }
        String id = id_pw[0];
        String pw = id_pw[1];
        String[] ids = new String[n];
        String[] pws = new String[n];
        for (int i = 0; i < n; i++) {
            ids[i] = db[i][0];
            pws[i] = db[i][1];
        }
        for (int i = 0; i < n; i++) {
            if (ids[i].equals(id)) {
                return "wrong pw";
            }
        }
        return "fail";
    }
}